src/log.rs

```diff
@@ -0,0 +1,34 @@
+use std::str::FromStr;
+use tracing::level_filters::LevelFilter;
+use tracing_subscriber::fmt::time::OffsetTime;
+
+const DEFAULT_LOG_LEVEL: LevelFilter = LevelFilter::INFO;
+
+pub fn init() {
+    let level = level_filter(option_env!("MOZART_LOG"));
+    let time = OffsetTime::local_rfc_3339().expect("could not initialize time offset");
+    tracing_subscriber::fmt()
```

Comment:
Guessing this is options for the logging mechanism?

Reply:
Yes, it determines the level of logging based on an environment variable or a default value.

Likewise it determines the local time and sets the offset, after which the subscriber is initialised with the supplied configuration.